exporter/exporterhelper/internal/producer_consumer_queue.go

```diff
@@ -1,32 +1,45 @@
 // Copyright The OpenTelemetry Authors
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//       http://www.apache.org/licenses/LICENSE-2.0
+// http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why this?

Reply:
That was the template used in `internal` so I adjusted. But I think we should actually use the format present elsewhere (with indentation)